vyxal/elements.py

```diff
@@ -561,11 +561,13 @@ def contains(lhs, rhs, ctx):
     """Element c
     (any, any) -> count of a in b
     """
-    lhs = iterable(lhs, ctx=ctx)
-    for item in lhs:
-        if item == rhs:
-            return 1
-    return 0
+    if list in vy_type(lhs, rhs, simple=True):
+        lhs = iterable(lhs, ctx=ctx)
+        for item in lhs:
+            if item == rhs:
+                return 1
+        return 0
```

Comment:
Just let `in` do the heavy lifting; it's completely equivalent.
```suggestion
        return int(lhs in rhs)
```

Reply:
If it accounted for type overloading and infinite lazylists, I would. But it doesn't.